packages/base-driver/lib/basedriver/core.ts

```diff
@@ -105,6 +105,8 @@ export class DriverCore<const C extends Constraints, Settings extends StringReco
 
   doesSupportBidi: boolean;
 
+  hostLastUsedByClient?: string;
```

Comment:
not sure if this should be a property of the all drivers or the umbrella driver only

Reply:
I was actually thinking about whether it would make sense to use https://nodejs.org/api/async_context.html#new-asynclocalstorage to store various request properties

We currently only use this approach for logger (too add request if to each log line), but might also extend it further